src/services/layerHelper.js

```diff
@@ -48,7 +48,7 @@ ngeo.LayerHelper.GROUP_KEY = 'groupName';
  * @param {string} sourceURL The source URL.
  * @param {string} sourceLayersName A dot separated names string.
  * @param {string=} opt_serverType Type of the server ("mapserver",
- *     "geoserver", qgisserver, …).
+ *     "geoserver", "qgis", …).
```

Comment:
It's qgisserver not qgis ..., see: http://mapfish.github.io/mapfish-print-doc/layers.html#Wms%20Layer
If somewere it's not like that it's an issue ...